src/utils/TodosContext.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  ReactNode,
+  createContext, useContext, useEffect, useState,
+} from 'react';
+import { Todo } from '../types/Todo';
+import { client } from './fetchClient';
+
+interface TodosContextInterface {
+  todos: Todo[];
+  setTodos(todos: Todo[]): void;
+  error: boolean;
+  setError(err: boolean): void;
+}
+
+export const TodosContext = createContext<TodosContextInterface>(
+  {
+    todos: [],
+    setTodos: () => { },
```

Comment:
don't leave empty functions
```suggestion
    setTodos,
```

Reply:
linter doesn't allow your option. I did it based on Volodymyr's examples.